src/odemis/driver/piezomotor.py

```diff
@@ -787,6 +861,80 @@ def _openSerialPort(port):
 
         return ser
 
+    @classmethod
+    def scan(cls):
+        """
+        returns (list of 2-tuple): name, kwargs
+        Note: it's obviously not advised to call this function if a device is already under use
+        """
+        # TODO: use serial.tools.list_ports.comports() (but only availabe in pySerial 2.6)
```

Comment:
You could definitely use this function now. We support Ubuntu 18.04+, which come with pySerial 3.4.

Reply:
What's the advantage of using this function over glob.glob? I just copied this comment from somewhere else.